src/app/core/services/application.service.ts

```diff
@@ -5,6 +5,7 @@ import { GithubRelease } from '../models/github/github.release';
 import { map } from 'rxjs/operators';
 
 const appSetting = require('../../../../package.json');
+export const currentVersion = appSetting.currentVersion;
```

Comment:
I think you meant `appSetting.version` ? (there's already a variable for it, too)
We can remove this though, as we can store the version info in the `AppConfig` (see my other comment)

Reply:
👍